resources/bundled_packages/cura.json

```diff
@@ -1714,7 +1732,7 @@
             "description": "Example package for material and quality profiles for Ultimaker materials.",
             "package_version": "1.4.0",
             "sdk_version": "8.6.0",
-            "website": "https://ultimaker.com/products/materials/cpe",
+            "website": "https://ultimaker.com/in/cura/materials/ultimaker-cpe-plus/printing-guidelines",
```

Comment:
Isn't this support website?
https://ultimaker.com/materials/s-series-cpe-plus/

Reply:
Correct, I'll change that.